packages/nouns-api/prisma/migrations/20221024145454_added_token_supply_on_create/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "Idea" ADD COLUMN     "tokenSupplyOnCreate" INTEGER DEFAULT 0;
```

Comment:
do you think we should maybe run a script that retroactively updates all existing ideas to have some "reasonable" tokenSupply so they are also able to show tags?

Reply:
I see your comment above -- sounds good.